js/performance.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+import * as msg from "gen/msg_generated";
+import { sendSync } from "./dispatch";
+import * as flatbuffers from "./flatbuffers";
+import { assert } from "./util";
+
+export const performance = {
```

Comment:
I think this should be an instance of a class called `Performance`, and `window.performance` would be effectively an instance of it.

```javascript
export class Performance {
  now(): number {
    // code
  }
}
```

And instantiate it in the [globals](https://github.com/denoland/deno/blob/master/js/globals.ts) as suggested by Ryan.

Reply:
Yes. Let’s do it in a new file js/performance.ts